File: Python/1015.py
# Problema 1015 - URI - Iniciante - Nível 1

# Adicionando "math" para usar  "sqrt" e "pow"
import math 
# Lendo x1 e y1
x1, y1 = input().split(" ")
# Lendo x2 e y2
x2, y2 = input().split(" ")
# Imprimindo a distancia
print(f'{(math.sqrt(math.pow((float(x2) - float(x1)), 2) + math.pow((float(y2) - float(y1)), 2))):.4f}')

File: Lua/Beginner/1015.lua
function distancia(x1, y1, x2, y2)
	dx = x1 - x2
	dy = y1 - y2
	return math.sqrt(dx^2 + dy^2)
end

x1, y1 = io.read('*n', '*n')
x2, y2 = io.read('*n', '*n')

print(string.format('%.4f', distancia(x1, y1, x2, y2)))